### PR TITLE
Dev sb bg exp

### PR DESCRIPTION
**Pull request by keeganbrown**

### CODE REVIEW

File: bin/dev [hunk 1]
```diff
@@ -2,6 +2,7 @@
 'use strict';
 const exec = require('child_process').exec;
 const commands = [
+  'rm ./brain/brain.json',
```
Comment (keeganbrown) [inline bin/dev:5]: uh test

**Comment by keeganbrown** [comment]
uh

**Comment by keeganbrown** [comment]
hye

File: bin/dev [hunk 2]
```diff
@@ -2,6 +2,7 @@
...
   'NODE_ENV=production webpack -p --progress --colors',
   'node ./server/server.js',
```
Comment (keeganbrown) [inline bin/dev:7]: test 2

**Comment by keeganbrown** [comment]
+1

**Comment by keeganbrown** [comment]
👍

**Comment by keeganbrown** [comment]
:shipit:

**Comment by keeganbrown** [comment]
LGTM

File: package.json [hunk 3]
```diff
@@ -4,36 +4,40 @@
...
     "convnetjs": "^0.3.0",
-    "es6-promise": "^3.1.2",
     "express": "^4.13.4",
-    "gsap": "^1.18.3",
+    "gsap": "^1.19.0",
+    "lodash": "^4.13.1",
     "redis": "^2.6.0-1",
     "script-loader": "^0.6.1",
     "sylvester": "0.0.21",
+    "twgl.js": "^1.7.1",
```
Comment (keeganbrown) [inline package.json:40]: bloop

Comment (keeganbrown) [inline package.json:40]: babasodijads;lkhrtghsrbsgr

File: bin/deploy [hunk 4]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+ssh node@generative-artist.smashingboxes.com 'cd learning-generative-art && git fetch --all && git reset --hard origin/dev-sb-bg-exp && npm run build'
```
Comment (joeyjoejoejr) [inline bin/deploy:2]: like this?

File: package.json (hunk 3 above)
Comment (keeganbrown) [inline package.json:40]: asdfasdfasd

**Pull request closed without merging**